tests/helpers.py

```diff
@@ -734,16 +734,16 @@ def get_worker_nodes():
     return worker_nodes_list
 
 
-def measure_pvc_creation_time(interface, pvc_name):
+def get_start_creation_time(interface, pvc_name):
     """
-    Measure PVC creation time based on logs
+    Get the starting creation time of a PVC based on provisioner logs
 
     Args:
         interface (str): The interface backed the PVC
         pvc_name (str): Name of the PVC for creation time measurement
 
     Returns:
-        float: Creation time for the PVC
+        datetime object: Creation time for the PVC
```

Comment:
datetime object: Start time of PVC creation

Reply:
done